components/create/CreateLanding.vue

```diff
@@ -131,6 +153,18 @@ const gotoPathAfterLogin = (path: RawLocation) => {
     }
   }
 
+  &-box-waifu {
+    width: 17rem;
+  }
+
+  &-content-waifu {
+    text-align: center;
+
+    &-text {
+      margin-top: 2rem;
+    }
```

Comment:
`mt-6`

Reply:
<img width="370" alt="Screen Shot 2023-10-18 at 9 53 02 PM" src="https://github.com/kodadot/nft-gallery/assets/39299315/285bab2b-ccc6-4313-8337-8ba6d26c9e97">
so this class is already there, I didn't add it, its suppose to be margin-top 0 for mobile thats why theres a separate class I think